src/deepsparse/yolo/pipelines.py

```diff
@@ -197,9 +203,25 @@ def process_inputs(
             iou_thres=inputs.iou_thres,
             conf_thres=inputs.conf_thres,
             multi_label=inputs.multi_label,
+            original_image_shapes=original_image_shapes,
         )
         return [image_batch], postprocessing_kwargs
 
+    def _scale_boxes(
```

Comment:
Since we are planning to have this function in multiple pipelines, let's make it public and put docstring on it

Reply:
Let's start by keeping this here, eventually when we move this out into a parent pipeline I think it would make sense to make it public, also a pipeline importing a function from another pipeline (at the same level) seems off. Are you suggesting to move this out of the `YOLOPipeline` class altogether and into something like utils?